frontend/src/views/DashboardPage/DashboardPage.utils.ts

```diff
@@ -75,6 +75,12 @@ export type FormData = yup.InferType<typeof schema>;
 export type TSecretDetailsOpen = { index: number; id: string };
 export type TSecOverwriteOpt = { secrets: Record<string, { comments: string[]; value: string }> };
 
+const formatVal = (val?: string) => {
+  if (!val) return "";
+  if (!val.match("\n")) return val;
+  return `"${val.replace(/\n/g, "\\n")}"`;
```

Comment:
What is the reason behind escaping new lines? Why was the issue with new lines in values?

Reply:
got it